src/Step/Deploy/SplitDbConnection.php

```diff
@@ -148,7 +151,8 @@ public function execute()
         if (!empty($missedSplitTypes)) {
             $this->logger->warning(
                 'Variable SPLIT_DB does not have data which were already split types: '
```

Comment:
Not sure what this means -- The SPLIT_DB variable is missing the names specifying the split connection types?
```suggestion
                'The SPLIT_DB variable is missing the configuration for split connection types: '
```